FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -26,9 +26,36 @@ using namespace xercesc;
 //<<<<<< MEMBER FUNCTION DEFINITIONS                                    >>>>>>
 
 namespace {
+
+  template< class CharType >
```

Comment:
That would make FWLite dependent upon Xerces. Having FWCore/Concurrency depend on Xerces was a bad idea originally since it added an unwanted dependency on other code.